README.md

```diff
@@ -1,6 +1,6 @@
 # sorbet-coerce
 
-Type coercion for [Sorbet](https://sorbet.org)
+> A type coercion lib works with [Sorbet](https://sorbet.org)'s static type checker and type definitions; raises an error if the coercion fails.
```

Comment:
```suggestion
A type coercion lib works with [Sorbet](https://sorbet.org)'s static type checker and type definitions; raises an error if the coercion fails.
```
I'd like to remove the `>` because it makes this paragraph a blockquote